server/item/crossbow.go

```diff
@@ -72,28 +71,22 @@ func (c Crossbow) ReleaseCharge(releaser Releaser, tx *world.Tx, ctx *UseContext
 
 	creative := releaser.GameMode().CreativeInventory()
 
-	rot := releaser.Rotation()
-	rot = cube.Rotation{-rot[0], -rot[1]}
-	if rot[0] > 180 {
-		rot[0] = 360 - rot[0]
-	}
-
 	dirVec := releaser.Rotation().Vec3().Normalize()
 
 	if firework, isFirework := c.Item.Item().(Firework); isFirework {
 		createFirework := tx.World().EntityRegistry().Config().Firework
 		fireworkEntity := createFirework(world.EntitySpawnOpts{
 			Position: torsoPosition(releaser),
 			Velocity: dirVec.Mul(1.5),
-			Rotation: rot,
+			Rotation: releaser.Rotation().Opposite(),
```

Comment:
Is this supposed to be `releaser.Rotation().Neg()`?

Reply:
yeah it is fixed